src/App.jsx

```diff
@@ -14,70 +16,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [select, selectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1
+        className="title is-flex is-align-items-center"
+      >
+        {select === ''
+          ? 'No goods selected'
+          : (
+            <>
+              {`${select} is selected`}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => selectedGood('')}
+              />
+            </>
+          )
+        }
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              className={good === select
+                ? 'has-background-success-light'
+                : ''}
+              key={good}
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  data-cy={good === select
```

Comment:
don't repeat yourself, create variable like `isSelectedGood` for `good === select`